internal/addon/var.go

```diff
@@ -31,9 +31,14 @@ const (
 	InstrumentationResource         = "instrumentations"
 	SpokeOTELColName                = "mcoa-instance"
 	SpokeInstrumentationName        = "mcoa-instance"
+	SpokeUIPluginName               = "monitoring"
 	SpokeOTELColNamespace           = "mcoa-opentelemetry"
 	otelColProbeKey                 = "replicas"
 	otelColProbePath                = ".spec.replicas"
+
+	UIPluginsResource = "uiplugins"
```

Comment:
Since this is not being used outside of this package we can make it private
```suggestion
	uiPluginsResource = "uiplugins"
```